syz-ci/manager.go

```diff
@@ -910,12 +913,35 @@ func (mgr *Manager) uploadCoverStat(fuzzingMinutes int) error {
 	go func() {
 		decoder := json.NewDecoder(resp.Body)
 		for decoder.More() {
-			var covInfo cover.CoverageInfo
-			if err := decoder.Decode(&covInfo); err != nil {
-				pw.CloseWithError(fmt.Errorf("failed to decode CoverageInfo: %w", err))
+			if err := f(pw, decoder, curTime); err != nil {
+				pw.CloseWithError(fmt.Errorf("callback: %w", err))
 				return
 			}
-			if err := cover.WriteCIJSONLine(pw, covInfo, cover.CIDetails{
+		}
+		pw.Close()
+	}()
+	fileName := fmt.Sprintf("%s/%s-%s-%d-%d.jsonl",
+		mgr.mgrcfg.DashboardClient,
+		mgr.name, curTime.Format(time.DateOnly),
+		curTime.Hour(), curTime.Minute())
```

Comment:
Nit: not that it matters much, but do we care about hours and minutes here? If there were several restarts on the same day, is it not okay to just overwrite the jsonl?

Reply:
It costs nothing and guarantees uniqueness. The goal was to generate the human readable, unique name.
But you're probably right, it is not a big deal.